CMakeLists.txt

```diff
@@ -48,22 +48,29 @@ SET (CMAKE_VERBOSE_MAKEFILE 0) # Use 1 for debugging, 0 for release
 #
 # Project Output Paths
 #
-#MESSAGE(${PROJECT_SOURCE_DIR})
-#SET (MAINFOLDER ${PROJECT_SOURCE_DIR})
 SET (LEMONADE_DIR ${PROJECT_SOURCE_DIR})
-#MESSAGE(${LEMONADE_DIR})
-SET (EXECUTABLE_OUTPUT_PATH "${LEMONADE_DIR}/build/bin")
-SET (LIBRARY_OUTPUT_PATH "${LEMONADE_DIR}/lib")
+
+#
+# Check if LEMONADE_DIR is given
+#
+if (DEFINED INSTALLDIR_LEMONADE)
+    message("INSTALLDIR_LEMONADE set to "${INSTALLDIR_LEMONADE})
+else()
+    SET (INSTALLDIR_LEMONADE "${LEMONADE_DIR}/build/")
+    message("INSTALLDIR_LEMONADE set automatically to "${INSTALLDIR_LEMONADE}". Use -DINSTALLDIR_LEMONADE=/path/to/install/LeMonADE/ for other destination")
+endif()
+
+SET (EXECUTABLE_OUTPUT_PATH "${INSTALLDIR_LEMONADE}/bin")
+SET (LIBRARY_OUTPUT_PATH "${INSTALLDIR_LEMONADE}/lib")
```

Comment:
Having lib within the INSTALLDIR_LEMONADE is a good idea to my opinion, as it allows to have more than one compiled version in different build directories.